app/views/users/show.html.erb

```diff
@@ -6,7 +6,7 @@
     <p><%= @user.bio %></p>
   </div>
 
-  <% @posts.each do |post| %>
+  <% @user.most_recent_posts.each do |post| %>
     <%= render 'layouts/custom/post_component', post: post %>
   <% end %>
   <!--button><%= link_to 'See all posts', user_posts_path(@user) %></#button-->
```

Comment:
## [OPTIONAL]
- [ ] Kindly fix this error also when the See all post button is clicked. 
![image](https://user-images.githubusercontent.com/65068771/161164703-f4ef78d3-d5f8-41a8-8ff1-c18a58bef456.png)